packages/frontend/src/constants.ts

```diff
@@ -33,50 +26,62 @@ export const DATA_CDN_URL = getServiceURL(
     __BUILDING_MODE__ === "production",
 );
 
-export interface AugmentedChain extends Chain {
-    logo: FunctionComponent<
-        SVGProps<SVGSVGElement> & { title?: string | undefined }
-    >;
-    iconBackgroundColor: string;
-    transport: Transport;
-}
-
-export const SUPPORTED_CHAINS =
+export const SUPPORTED_CHAINS: [Chain, ...Chain[]] =
     __BUILDING_MODE__ === "production"
-        ? ([
+        ? [
               {
                   ...gnosis,
-                  logo: GnosisLogo,
-                  iconBackgroundColor: "#04795b",
-                  transport: http(),
-              },
-          ] as const)
-        : ([
+                  rpcUrls: {
+                      default: {
+                          http: ["https://rpc.ankr.com/gnosis"],
+                      },
+                  },
+              } as const satisfies Chain,
+          ]
+        : [
               {
-                  ...sepolia,
-                  logo: EthereumLogo,
-                  iconBackgroundColor: "#8637ea",
-                  transport: http(),
-              },
+                  ...polygonMumbai,
+                  rpcUrls: {
+                      default: {
+                          http: ["https://polygon-mumbai-pokt.nodies.app/"],
+                      },
+                  },
+              } as const satisfies Chain,
+              sepolia,
               {
                   ...scrollSepolia,
-                  logo: ScrollLogo,
-                  iconBackgroundColor: "#213147",
                   blockExplorers: {
                       default: {
                           name: "Scrollscan",
                           url: "https://sepolia.scrollscan.com",
                       },
                   },
-                  transport: http(),
-              },
-              {
-                  ...polygonMumbai,
-                  logo: PolygonLogo,
-                  iconBackgroundColor: "#8247E5",
-                  transport: http(),
-              },
-          ] as const);
+                  rpcUrls: {
+                      default: {
+                          http: ["https://sepolia-rpc.scroll.io"],
+                      },
+                  },
+              } as const satisfies Chain,
+          ];
+
+export const SUPPORTED_CHAIN_ICONS = {
+    [gnosis.id as number]: {
+        logo: GnosisLogo,
+        iconBackgroundColor: "#04795b",
```

Comment:
We can probably just call this `backgroundColor` or `bgColor`. Also, we should find a way in Typescript to force an error if not all the chain ids from `SUPPORTED_CHAINS` are specified (it would cause a missing icon in the frontend).